build/release/README.md

```diff
@@ -55,6 +55,47 @@ The tags allow for a progression of pre-releases such as:
 
 The release tags should be agreed on by the release team.
 
+## Bulk backporting master changes to a release branch
+During the short weeks leading up to release, the release branch will exist with betas released
+periodically. Betas will generally include all commits in master; that is, all commits in master are
+likely to be backported to the release branch. Here are instructions for how to bulk backport the
+master branch's commits to a release branch.
+
+Let's parameterize these instructions slightly
+```sh
+RELEASE_BRANCH='release-1.2'
+```
+
+The following views may be useful. Open them side-by-side.
+- **A :** `git log --format=oneline --no-merges --topo-order $RELEASE_BRANCH`
+- **B :** `git log --format=oneline --no-merges --topo-order master`
+
+Open a new window with the following view to list the commits which are part of the `master` branch
+and not part of the release branch. These will be in alphabetical sorted order rather than commit
+topological order.
+- **C :** `comm -1 -3 <(git log --format=format:'%s' --no-merges --topo-order release-1.2 | sort) <(git log --format=format:'%s' --no-merges --topo-order master | sort)`
+
+Open a fourth window to be our working window.
+1. `git co master`
+2. `git co -b working-branch-for-$RELEASE_BRANCH` to make a working branch equivalent with `master`
+3. `git rebase --interactive $RELEASE_BRANCH` to rebase `master` changes on *top* of the release branch
```

Comment:
When I tried the `git rebase --interactive` command last week it seemed that something wasn't quite right and the commit for the beta.0 tag in the manifests gave me a merge conflict. Then I fell back to do the bulk backport with these steps:
1. `git fetch --all`
1. `git checkout -b bulk-backport upstream/master`
1. `git push origin bulk-backport`
1. Open a PR to merge the branch to `release-1.2` branch
This didn't give me any merge conflicts and seemed like a simple way to get master back in sync with the release branch. Is there some advantage to the interactive rebase that I'm missing?

Reply:
I dunno. I don't think GitHub writes their own custom logic for PRs. I suspect that there would be the same double-commit issue I saw if I were to have done what you suggest, but I'm not sure. I can't speculate about any of this.

Reply:
This is what happens if I attempt this strategy. I see a number of commits it's trying to add including the 2 rgw commits:
![image](https://user-images.githubusercontent.com/3989526/71200201-d9821680-2254-11ea-9264-6dfa8bb9d7d4.png)

Reply:
hmm I wonder what's up with those two commits? I can't anything special about them